website/src/plugins/examples/index.js

```diff
@@ -12,11 +12,13 @@ function all_examples_for_type(folder, type) {
   let examples = [];
   let dir = path.join(__dirname, `../../../docs/${folder}/${type}`);
   fs.readdirSync(dir).forEach(function (file) {
-    let name = file.split(".").slice(0, -1).join(".");
-    let data = fs.readFileSync(path.join(dir, file));
-    const { frontMatter } = parseMarkdownString(data);
-    frontMatter["name"] = name;
-    examples.push(frontMatter);
+    if (file.endsWith(".md")) {
```

Comment:
Originally, this iterated over not only files, but also folders that `jupyter nbconvert` generates for supporting images in generated markdown files. 

However, we only need to iterate over markdown files, so I made this change.

Reply:
If you replace the images with url links, this won't be needed anymore. But pls feel free to keep it if you prefer.